codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,13 +175,19 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
+        FlowState state;
```

Comment:
@buchgr - Why can you not make this final?  It is only set 1 time.  The reason to make it final is because you are initializing it in the `try {..}` block and using it after.  If you don't set it in the block and use it after the block the compiler will complain.